ui/components/app/asset-list/asset-list.js

```diff
@@ -95,66 +83,12 @@ const AssetList = ({ onClickAsset }) => {
     getIstokenDetectionInactiveOnNonMainnetSupportedNetwork,
   );
 
-  const contractExchangeRates = useSelector(
```

Comment:
All of this code was essentially moved to the new `ui/hooks/useAccountTotalFiatBalance.js` hook.